test/framework/setup.go

```diff
@@ -180,6 +180,26 @@ spec:
 	return YamlK8s(mesh)
 }
 
+func MTLSMeshWithMeshServicesAndEgressKubernetes(name, meshServicesEnabled string, egress bool) InstallFunc {
```

Comment:
Could we please implement this one
https://github.com/kumahq/kuma/issues/11294

Reply:
after merge https://github.com/kumahq/kuma/pull/11448 I can change

Reply:
it's merged